counsel.el

```diff
@@ -2505,6 +2505,12 @@ INITIAL-DIRECTORY, if non-nil, is used as the root directory for search."
               :keymap counsel-find-file-map
               :caller 'counsel-file-jump)))
 
+(ivy-set-actions
+ 'counsel-file-jump
+ '(("d" (lambda (x)
+          (dired (or (file-name-directory x) default-directory)))
```

Comment:
Please don't quote lambdas. Either evaluate it like so:

```el
`(("d" ,(lambda (x) ...) ...))
```

or define it via `defun` first. Thanks.

Reply:
Done, thanks!